### PR TITLE
feat: enable builds with Fedora 39

### PR DESCRIPTION
**Pull request by p5**
An experiment to see how much we need to change for Fedora 39.
The images are now being pushed to Quay for base and silverblue.

I am not expecting many packages to build.  This is just an experiment.

### CODE REVIEW

**Review by bsherman — APPROVED** [review]

**Review by bsherman — DISMISSED** [review]

**Comment by KyleGospo** [comment]
Hopefully just a matter of negativo17 and rpmfusion supporting 39

**Comment by bsherman** [comment]
> Hopefully just a matter of negativo17 and rpmfusion supporting 39

Yup. Though, I'd like to move towards only depending on fedora upstream, rpmfusion, and COPRs which we have some control over, if possible.

**Review by EyeCantCU — APPROVED** [review]

File: Containerfile.common [hunk 1]
```diff
@@ -36,9 +35,14 @@ RUN /tmp/build-kmod-ryzen-smu.sh
 RUN /tmp/build-kmod-steamdeck.sh
 RUN /tmp/build-kmod-v4l2loopback.sh
 RUN /tmp/build-kmod-wl.sh
-RUN /tmp/build-kmod-xpadneo.sh
-RUN /tmp/build-kmod-xpad-noone.sh
-RUN /tmp/build-kmod-xone.sh
+
+# Exclude negativo17 kmods from Fedora 39
+RUN if grep -q ${FEDORA_MAJOR_VERSION}; then \
```
Comment (p5) [inline Containerfile.common]: Probably need something like this to run on F38 and lower
```
if [ "${FEDORA_MAJOR_VERSION}" -lt 39 ]; then
```

File: Containerfile.common [hunk 2]
```diff
@@ -37,7 +37,7 @@ RUN /tmp/build-kmod-v4l2loopback.sh
 RUN /tmp/build-kmod-wl.sh
 
 # Exclude negativo17 kmods from Fedora 39
-RUN if grep -q ${FEDORA_MAJOR_VERSION}; then \
+RUN if grep -qv "39" <<< ${FEDORA_MAJOR_VERSION}; then \
```
Comment (p5) [inline Containerfile.common:40]: LGTM - but I don't know grep...  

More like looks good to GH Copilot :)

Comment (EyeCantCU) [reply Containerfile.common:40]: We use the same implementation over in Bazzite for checking image details :). Should be good!

**Pull request merged**